### PR TITLE
add a recipe for perl-gd-svg

### PR DESCRIPTION
**Pull request by Xiangyang1984**
add a recipe for perl-gd-svg

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by Xiangyang1984** [comment]
ping ``@bioconda/core`.

**Comment by Xiangyang1984** [comment]
ping @bioconda/core

**Comment by Xiangyang1984** [comment]
ping @bioconda/core

**Comment by Xiangyang1984** [comment]
ping @bioconda

**Comment by Xiangyang1984** [comment]
@BiocondaBot

**Comment by Xiangyang1984** [comment]
please review & merge

**Comment by Xiangyang1984** [comment]
@22noon

**Comment by Xiangyang1984** [comment]
@22noon  please review & merge

**Comment by Xiangyang1984** [comment]
@BiocondaBot please add label

**Comment by Xiangyang1984** [comment]
@BiocondaBot please merge

**Comment by Xiangyang1984** [comment]
@BiocondaBot please merge

**Comment by Xiangyang1984** [comment]
@BiocondaBot please merge

**Comment by Xiangyang1984** [comment]
@BiocondaBot please review & merge

**Comment by Xiangyang1984** [comment]
@BiocondaBot please update

**Comment by Xiangyang1984** [comment]
@BiocondaBot please fetch artifacts

**Comment by dpryan79** [comment]
Do you need this as a dependency for something else you want to put in bioconda?

**Comment by Xiangyang1984** [comment]
@dpryan79 we needs this as a dependency for our package，which we want to put in bioconda

File: recipes/perl-gd-svg/meta.yaml [hunk 1]
```diff
@@ -11,7 +10,6 @@ source:
 
 build:
   number: 0
-  noarch: generic
```
Comment (dpryan79) [inline recipes/perl-gd-svg/meta.yaml]: Isn't this correct for perl packages?

Comment (mbargull) [reply recipes/perl-gd-svg/meta.yaml]: Nvm; had a wrong thought.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Comment by Xiangyang1984** [comment]
@dpryan79 thanks for you help, I want to known why there are no linux- and mac- version of perl-gd-svg build in bioconda?

**Comment by dpryan79** [comment]
There don't need to be OS-specific builds, the same package works on all platforms.

**Comment by Xiangyang1984** [comment]
@dpryan79 Ok, very thanks for your help. By the way, can I join the bioconda organization?

**Pull request merged**